index.html

```diff
@@ -38,35 +39,220 @@ <h1 class="name">I’m Abraha Glad to see you!</h1>
           you see and have a project you need coded, don’t hestiate to contact
           me.
         </p>
-        <div class="contact-message">
-          <a href="#">LET'S CONNECT</a>
+        <div class="message-wrap">
+          <div class="contact-message">
+            <a href="#">LET'S CONNECT</a>
+          </div>
+          <div class="social-media">
+            <ul>
+              <li>
+                <a href="https://twitter.com/Alamsbut"
+                  ><i class="fa fa-twitter fa-lg"></i
+                ></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/abraha-kahsay-492771135/"
+                  ><i class="fa fa-linkedin fa-lg"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-medium fa-lg"></i></a>
+              </li>
+              <li>
+                <a href="https://github.com/AbrahaKahsay"
+                  ><i class="fa fa-github fa-lg"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-angellist fa-lg"></i></a>
+              </li>
+            </ul>
+          </div>
         </div>
-        <div class="social-media">
-          <ul>
-            <li>
-              <a href="https://twitter.com/Alamsbut"
-                ><i class="fa fa-twitter fa-lg"></i
-              ></a>
-            </li>
-            <li>
-              <a href="https://www.linkedin.com/in/abraha-kahsay-492771135/"
-                ><i class="fa fa-linkedin fa-lg"></i
-              ></a>
-            </li>
-            <li>
-              <a href="#"><i class="fa fa-medium fa-lg"></i></a>
-            </li>
+      </div>
+    </section>
+    <!-- work-section -->
+    <section class="works-section">
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements, you must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the templates in Figma. Please fix the wider margin on the right side of the page to match the original

### Your Version
![stories2](https://user-images.githubusercontent.com/63964646/149623503-da3bcaa9-7436-42d5-97c6-c6289d87f8d0.PNG)

### Original Version
![stories1](https://user-images.githubusercontent.com/63964646/149623488-9955e204-fb43-442a-9444-c970d5027444.PNG)

Reply:
- [ ] [REQUIRED] Kindly implement this as well. Please ensure you have equal margins on the right and left side of your cards as it is in the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![1](https://user-images.githubusercontent.com/58692590/149634976-73009ab3-1b28-4fef-99d6-bb26f1751f7f.png) | ![11](https://user-images.githubusercontent.com/58692590/149634980-547af750-ff4f-4229-9fe7-7e997386e8a5.png)